geodocker/geodocker-jupyter-geopyspark
# Add Mapnik

https://github.com/geotrellis/geonotebook/pull/13 does not work in jupyterhub, must use jupyter: `jupyter notebook --port=8000`.

To Test
   - Make sure that all bootstrap containers are up to date
   - Build from https://github.com/jamesmcclain/geodocker-jupyter-geopyspark/tree/feature/mapnik-wip
      - `docker run -it --rm --host=net quay.io/jupyter-geopyspark:mapnik bash`
      - `jupyter notebook --port=8000` from inside of container
      - Test that KitWare notebooks work (one is now included in this repository)
   - Build from this branch
      - Use as before; confirm that everything that used to work still does

![screenshot from 2017-08-18 19 52 47](https://user-images.githubusercontent.com/11281373/29496108-de6655f8-8599-11e7-8f3b-5b70d474182a.png)
![screenshot from 2017-08-20 11 12 47](https://user-images.githubusercontent.com/11281373/29496107-de65d4ac-8599-11e7-8b12-504081949a02.png)

Fixes geodocker/geodocker-jupyter-geopyspark#18
